Document Update (#27694)
grafana/grafana: docs/sources/installation/upgrading.md

will keep working with unencrypted passwords. If you want to migrate to encrypted storage for your existing data sources
you can do that by:

> - For data sources created through UI, you need to go to data source config, reenter the password or basic auth

Reviewer: How about "re-enter"?

### Session storage is no longer used

In 6.2 we completely removed the backend session storage since we replaced the previous login session implementation with an auth token.

> If you are using Auth proxy with LDAP a shared cached is used in Grafana so you might want configure [remote_cache] instead. If not
> Grafana will fallback to using the database as a shared cache.

Reviewer 2: ```suggestion
If you are using Auth proxy with LDAP, a shared cached is used in Grafana, so you might want to configure [remote_cache] instead. If not,
Grafana will fallback to using the database as a shared cache.
```

> Grafana will fallback to using the database as a shared cache.

Reviewer: Use "fall back"

Reviewer 3: Agreed. "Fall back" is a verb, "fallback" is a noun. It's similar to "set up" and "setup."

Reviewer: Can you change "fallback" to "fall back"? Thank you.

Author: I have made the changes. Thanks for the suggestion @achatterjee-grafana

### Cookie path

> Starting from Grafana v7.0.0, the cookie path does not include the trailing slash if Grafana is served from a subpath in order to align with [RFC 6265](https://tools.ietf.org/html/rfc6265#section-5.1.4). However, stale session cookies (set before the upgrade) can result in unsuccessful logins because they cannot be deleted during the standard login phase due to the changed cookie path. Therefore users experiencing login problems are advised to manually delete old session cookies, or administrators can fix this for all users by changing the [`login_cookie_name`]({{< relref "../administration/#login-cookie-name" >}}), so the old cookie would get ignored.

Reviewer 2: I would revert this change, since it's completely subjective (i.e., the original wording is just as correct).

Author: I feel, both looks same and no changes present.

Reviewer 2: There is a change from "can not" to "cannot".

Author: I have reverted the changes.

For existing alert notification channels, there is no automatic migration of storing sensitive settings encrypted, and they will continue to work as before. Migration must be done manually. Opening a configured alert notification channel in the UI and saving it will store sensitive settings encrypted and at the same time reset the historic unencrypted setting of that alert notification channel in the database.

> > Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and as a result not function as expected.

Reviewer: Use "and, as a result,"

> > Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and by that it does not function as expected.

Reviewer 2: ```suggestion
> Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and as a result not function as expected.
```
